examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -35,34 +35,34 @@ export const LayoutDefinition = {
 };
 
 // Type definition for `core::byte_array::ByteArray` struct
-export interface ByteArray {
+export type ByteArray = {
     data: String[];
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)